dist/test_tag.py

```diff
@@ -0,0 +1,203 @@
+import os
+import sys
+
+##### LOCAL VARIABLES AND CONSTANTS #####
+component = ""
+testingArea = ""
```

Comment:
To be consistent with the variable naming convention in other python files under dist/ can we use _ to separate words instead of camel case. So testing_area, etc.